src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public enum Color {
+    RED("red"),
+    GREEN("green"),
+    BLUE("blue"),
+    YELLOW("yellow"),
+    BLACK("black"),
+    WHITE("white");
```

Comment:
not a mistake but I don't see much point in having the same colorname as enum value but in lowercase, we can call .lowercase() on enum.name()